_test/test_transformation/test_ortho_axes.m

```diff
@@ -625,7 +625,11 @@ function test_get_bin_centers_2D_4d(~)
                 [dbr(1,3),dbr(2,3)];[dbr(1,4),1,dbr(2,4)]};
             ab = ortho_axes(bin0{:});
 
-            [nodes,en,nbins] = ab.get_bin_nodes('-bin_centre',(dbr(2,:)-dbr(1,:))'/10);
+            char_size_des = (dbr(2,:)-dbr(1,:))/10;
+            char_size_ex  = (ab.img_range(2,:)-ab.img_range(1,:))./ab.nbins_all_dims;
+            mult = ceil(char_size_ex./char_size_des);
+            mult(mult<1) = 1;
+            [nodes,en,nbins] = ab.get_bin_nodes('-bin_centre',mult);
```

Comment:
this code from L.628 is highly opaque. Can you provide a description of what it is, in particular how it improves on the old dbr-dr/10